modules/pixi/PixiLayerCustomData.js

```diff
@@ -9,7 +9,7 @@ import { PixiFeaturePoint } from './PixiFeaturePoint';
 import { PixiFeaturePolygon } from './PixiFeaturePolygon';
 import { utilFetchResponse } from '../util';
 
-const CUSTOM_COLOR = 0x00ffff;
+const CUSTOM_COLOR = '#c6eb34'; // don't hard code this :)
```

Comment:
oh this is one of those changes from the colorpicker branch, right? We should probably just remove this change.

Reply:
Yep! I'll remove this